generator/client/msw-handlers.ts

```diff
@@ -168,10 +161,13 @@ export function generateMSWHandlers(spec: OpenAPIV3.Document) {
 
         const { path, query } = params
 
-        const { body, bodyErr } = bodySchema
-          ? validateBody(bodySchema, await req.json())
-          : { body: undefined, bodyErr: undefined };
-        if (bodyErr) return json(bodyErr, { status: 400 });
```

Comment:
`bodyErr` is already a `Response`, so we were seeing `{}` as the response body in the browser because `JSON.stringify(new Response()) == '{}'`.